webapp/graphite/util.py

```diff
@@ -72,13 +72,20 @@ def getProfileByUsername(username):
   except Profile.DoesNotExist:
     return None
 
-
 def is_local_interface(host):
+  is_ipv6 = False
   if ':' in host:
-    host = host.split(':',1)[0]
+    try:
+      socket.inet_pton(socket.AF_INET6, host)
+      is_ipv6 = True
```

Comment:
Ok, I'm trusting you here since I have little practical ipv6 experience to compare against. :smile_cat: